modules/menu/main/index.coffee

```diff
@@ -287,31 +287,30 @@ class Menu extends hx.EventEmitter
           checkEvent(e, self)
         self.emit 'keydown', e
 
-      self.dropdown._.dropdown
-        .on 'click', 'hx.menu', (e) ->
-          # get the closest menu item - uses nodes as blank selection can be
-          # returned if the target is a hx-menu-item
-
-          target = if hx.select(e.target).classed('hx-menu-link')
-            e.target
-          else
-            hx.select(e.target).closest('.hx-menu-item').node()
-
-          if target
-
-            index = -1
-
-            t = hx.select(target)
-            if t.classed('hx-menu-link')
-              allItems = getAllItems(self)
-              i = 0
-              while index < 0 and i < allItems.length
-                if allItems[i].node is target
-                  index = i
-                  break
-                i += 1
-              setActive(self, index, undefined, true)
-            targetElem.node().focus()
+      self.dropdown._.dropdown?.on 'click', 'hx.menu', (e) ->
```

Comment:
Pretty sure it was because I'd added an autocomplete to a button so it wasn't passing an autocomplete check and hiding the dropdown before this was being called, pretty sure it was caused by my poor use of autocomplete on a button rather than this code but I figured it should probably have the check as all the other uses check for its existence as well :)